server/config/subject_page.proto

```diff
@@ -182,15 +182,17 @@ message BarTileSpec {
   bool horizontal = 4;
   // maximum number of places to display
   int32 max_places = 5;
+  // maximum number of variables to display
+  int32 max_variables = 6;
   // bar sort order
```

Comment:
nit: not that it matters here, but its generally good practice to not change existing tag numbers :)